src/main/python/TravelTimeReporter.py

```diff
@@ -392,7 +392,7 @@ def get_ff_time(self, nev = False):
                 np.maximum(
                     self.constants[flavor + "DiversionConstant"] + direct_time,
                     self.constants[flavor + "DiversionFactor"] * direct_time
-                ) + self.constants[flavor + "WaitTime"],
+                )
```

Comment:
Still need to include comma here for np.maximum arguments

Reply:
np.where arguments, not np.maximum

Reply:
Thanks! I have fixed it.

Reply:
Comma needs to be before self.settings["infinity"]